4337/src/deploy/entrypoint.ts

```diff
@@ -0,0 +1,33 @@
+import EntryPoint from '@account-abstraction/contracts/artifacts/EntryPoint.json'
+import { DeployFunction } from 'hardhat-deploy/types'
+
+const ENTRY_POINT = process.env.DEPLOY_ENTRY_POINT
```

Comment:
Is it me, or the env var name is a little bit misleading? I'd expect it to be a boolean to deploy an entry point, but it seems like a fallback address if the entry point is not deployed.

Reply:
Note that this was the environment variable before this PR. I _think_ the rationale behind it was that all environment variables for the `runOp` script were prefixed with `SCRIPT_*` and all the environment variables for deployment were prefixed with `DEPLOY_*`, but agree that it is confusing and sounds like a true/false variable for “deploy entry point?”

Do you think `DEPLOYMENT_ENTRY_POINT` / `DEPLOYMENT_OVERRIDE_ENTRY_POINT` / `DEPLOYMENT_ENTRY_POINT_ADDRESS` would be clearer?

Reply:
Renamed to `DEPLOYMENT_ENTRY_POINT_ADDRESS`

Reply:
legend